### ISSUE TITLE
android build error (cannot find symbol [f]) 

### ISSUE THREAD

**Issue by flyingmate** [marcshilling/react-native-image-picker#75]
/Users/james/Dropbox/Development/react/Builder/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java:333: error: cannot find symbol
            f.createNewFile();
            ^
  symbol:   variable f
  location: class ImagePickerModule
/Users/james/Dropbox/Development/react/Builder/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java:339: error: cannot find symbol
            fo = new FileOutputStream(f);
                                      ^
  symbol:   variable f
  location: class ImagePickerModule
/Users/james/Dropbox/Development/react/Builder/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java:348: error: cannot find symbol
        return f.getAbsolutePath();
               ^
  symbol:   variable f
  location: class ImagePickerModule

**Comment by yfuks** [comment]
my bad, in ImagePickerModule.java line 328
replace 
```java
File file = new File(path, filname +".jpg");
```

into 
```java
File f = new File(path, filname +".jpg");
```

**Comment by flyingmate** [comment]
Thank you!

Please close this after publishing

**Comment by yfuks** [comment]
fix with 0.11.3
